src/components/game/container.js

```diff
@@ -0,0 +1,26 @@
+import { connect } from 'react-redux';
+
+import { selectSquare, goToMove, changeMovesOrder } from '../../actions';
+
+import Game from './game';
+
+const mapStateToProps = (state) => state;
+
+const mapDispatchToProps = (dispatch) => ({
+    onClick: (id) => {
```

Comment:
No need to create a complex function here.
update this one and below to `function: (param) => dipatch(dispatchFunction(param)),`
i.e. `onClick: (id) => dispatch(selectSquare(id)),`